client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -81,10 +83,61 @@ describe('FeedbackGeneratorService', function() {
             c: 'j'
           })
         ).toEqual('{"a": 3, "b": 5, "c": "j"}');
+        // Need to define and call blankFunction for 100% karma coverage
+        var blankFunction = function() {
+          return null;
+        };
+        blankFunction();
+        expect(
+          FeedbackGeneratorService._jsToHumanReadable(blankFunction)
+        ).toEqual('[UNKNOWN OBJECT]');
       }
     );
   });
 
+  describe('_getCorrectnessTestFeedback', function() {
+    it('should return output of user function with issues', function() {
```

Comment:
should return feedback on a user function that doesn't pass all correctness tests